src/browsergym/workarena/tasks/incident.py

```diff
@@ -0,0 +1,372 @@
+import json
+from typing import Any, Dict, List, Tuple
+
+import playwright.sync_api
+import requests
+
+from ..api.utils import HTTPError, db_delete_from_table, table_api_call
+from ..config import (
+    CREATE_INTERACTION_CONFIG_PATH,
+)
+from .base import AbstractServiceNowTask
+
+
+class ServiceNowIncidentTask(AbstractServiceNowTask):
+
+    def __init__(self, seed: int, fixed_config: Dict[str, Any] = None, start_rel_url: str = "/now/nav/ui/home") -> None:
+        super().__init__(seed, start_rel_url=start_rel_url)
+        self.task_is_setup = False
+        self.config = fixed_config if fixed_config else self.random.choice(self.all_configs())
+        self.timeout = 60000
+        self.created_sysids = []
+
+    def setup_goal(self, page: playwright.sync_api.Page) -> Tuple[str, dict]:
+        """Setup the task configuration and produce the goal."""
+
+        goal = self.config["goal"]
+        info = self.config
+
+        return goal, info
+
+    def cheat(self, page: playwright.sync_api.Page, chat_messages: List[str]) -> None:
+        pass
+
+    def all_configs(self):
+        raise NotImplementedError
```

Comment:
for these **create** incidents we should just reuse what exists. I will do a PR for these incident tasks